File: crudview-admin-sb/DataAccess/ModelsMetaData/DashboardReport.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DataAccess.Core.Models
{
    public class DashboardReport
    {
        public int Products { get; set; }
        public decimal Sales { get; set; }
        public decimal Discount { get; set; }
        public decimal ReceivedPayment { get; set; }
        public decimal DuePayment { get; set; }
    }
}

File: crudview-admin-celetial/App/wwwroot/js/models/DashboardReport.js
function DashboardReport(data) {
    var instance = this;
    instance.Products = ko.observable(data.products);
    instance.Sales = ko.observable(data.sales);
    instance.Discount = ko.observable(data.discount);
    instance.ReceivedPayment = ko.observable(data.receivedPayment);
    instance.DuePayment = ko.observable(data.duePayment);
}